spec/support/authenticated_session_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module AuthenticatedSessionHelper
+  def use_authenticated_current_user(options = {})
+    klass = options[:klass] || ApplicationController
```

Comment:
I took a different approach to this in adding auth for claim status
https://github.com/department-of-veterans-affairs/vets-api/pull/322/files#diff-d2e878440cb50adc721d22b2524e3365R12
My opinion is that we shouldn't be stubbing these methods in a request spec as they are meant to go through the whole request process, and instead we should set up a session and pass the appropriate headers.